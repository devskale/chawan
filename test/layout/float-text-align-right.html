<div style="text-align: right">
<div style="float: right; background-color: red">float</div>
<div style="float: left; background-color: red">float float float</div>
test
