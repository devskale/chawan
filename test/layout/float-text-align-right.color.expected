[48;2;255;0;0mfloat float float[49m                                                      test[48;2;255;0;0mfloat[49m
